scripts/system/create/edit.js

```diff
@@ -2923,4 +2925,17 @@ function getParentState(id) {
     return state;
 }
 
+function getDomainOnlyChildrenIDs(id) {
+    var allChildren = Entities.getChildrenIDs(id);
+    var realChildren = [];
+    var properties;
+    for (var i = 0; i < allChildren.length; i++) {
+        properties = Entities.getEntityProperties(allChildren[i], ["name"]);
+        if (properties.name !== entityShapeVisualizerSessionName && properties.name !== undefined) {
```

Comment:
```suggestion
        if (properties.name !== undefined && properties.name !== entityShapeVisualizerSessionName) {
```
Thought it doesn't matter in this code, I suggest testing against `undefined` first, just as a matter of good habit.